pkg/k8shandler/fluentd.go

```diff
@@ -400,6 +438,13 @@ func newFluentdPodSpec(cluster *logging.ClusterLogging, trustedCABundleCM *v1.Co
 		{Name: metricsVolumeName, MountPath: "/etc/fluent/metrics"},
 	}
 
+	exporterContainer.VolumeMounts = []v1.VolumeMount{
+		{Name: logVolumeMountName, MountPath: logVolumePath},
+		{Name: CrtKeyVolumeMountName, MountPath: CrtKeyVolumePath},
```

Comment:
change to "CrtKeyVolumeMountName" to "metricsVolumeName" and "CrtKeyVolumePath" into "/etc/fluent/metrics" (maybe best will be to put that into a variable and change In both)

Reply:
resolved